Gemfile

```diff
@@ -34,6 +34,7 @@ group :development, :test do
   gem "listen"
   gem "pry-byebug"
   gem "rspec-rails"
+  gem "rubocop", "~> 1.35.0", require: false
```

Comment:
We shouldn't need this line as rubocop-govuk already fixes rubocop to 1.35 (https://github.com/alphagov/finder-frontend/blob/de54facae7404ba618a3c8bb00d64d8ead6f041a/Gemfile.lock#L366) 

As far as I can tell, this line isn't actually doing anything despite what the commit message suggests 😕 

I think instead it'd need a pin for stringio and it doesn't look like there is one. If we do need to do that we should add a comment to explain why (as per https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#gemfile-organisation)

Reply:
I'm not going to pretend I understand any of this in any depth, although I tested this by putting this change up (tests passed) then adding a following commit to remove it (tests failed) so... yeah, dunno. However I think you've hit the solution in your second comment...

Reply:
Sure, I think what happened is that you managed to get a change to the Gemfile.lock in (old version downgrading stringio) at the same time as the rubocop-govuk change.